eng/Versions.props

```diff
@@ -2,7 +2,7 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.14.0</VersionPrefix>
+    <VersionPrefix>17.14.2</VersionPrefix>
```

Comment:
Don't do this, we haven't gone to final branding yet.

Reply:
@rainersigwald  please merge it to bypass the policy